tomcat7/image.yaml

```diff
@@ -66,6 +66,9 @@ artifacts:
     - path: jws-application-servers-3.1.1-RHEL7-x86_64.zip
```

Comment:
Is 3.1.1 required to apply 3.1.2?

Reply:
From the information I have, this is a cumulative patch which means 3.1.1 is not required, so I made the necessary changes.